app/src/screens/MapScreen.js

```diff
@@ -93,11 +94,16 @@ function MapScreen(): React.Node {
   const [filterMaxRssi, setFilterMaxRssi] = useState<string>('');
   const [filterMinHeight, setFilterMinHeight] = useState<string>('10');
   const [filterMaxHeight, setFilterMaxHeight] = useState<string>('');
-  const [cookies, setCookie] = useCookies(['token']);
+  const [token, setToken] = useState<?string>(() => {
+    const mapboxToken =
+      new URLSearchParams(window.location.search).get('access_token') ??
+      localStorage.getItem(TOKEN_KEY);
+    return mapboxToken;
+  });
 
-  const mapboxToken =
-    new URLSearchParams(window.location.search).get('access_token') ??
-    cookies.token;
+  const [tokenDialogOpen, setTokenDialogOpen] = useState<boolean>(() => {
+    return localStorage.getItem(TOKEN_KEY) == null;
+  });
```

Comment:
Same here, we can shorten the arrow function

Question though, do we want to open this popup even if there's a token in the URL? (And if so, should we maybe prefill the field with that info?)

```
() => return localStorage.getItem(TOKEN_KEY) == null
```